chroma_core/lib/service_config.py

```diff
@@ -991,9 +958,6 @@ def usage():
     elif command == "restart":
         service_config.stop()
         service_config.start()
-    elif command == "bundle":
```

Comment:
There used to be a `bundle` command here that poked at the bundles in the DB. Do we want to add a repo command so users can do CRUD over new repos?